src/storage/txn/commands.rs

```diff
@@ -350,6 +390,25 @@ command! {
 impl CommandExt for Prewrite {
     tag!(prewrite);
     ts!(start_ts);
+
+    fn write_bytes(&self) -> usize {
+        let mut bytes = 0;
+        for m in &self.mutations {
+            match *m {
+                Mutation::Put((ref key, ref value)) | Mutation::Insert((ref key, ref value)) => {
+                    bytes += key.as_encoded().len();
+                    bytes += value.len();
+                }
+                Mutation::Delete(ref key) | Mutation::Lock(ref key) => {
+                    bytes += key.as_encoded().len();
+                }
+                Mutation::CheckNotExists(_) => (),
+            }
+        }
+        bytes
+    }
+
+    gen_lock!(mutations, multiple, (|x| x.key()));
```

Comment:
Does anyone know how to remove the parentheses around the closure here?
The comiler gives me a warning abount this, but remove them directly will cause an error.

Reply:
Maybe remove the comma after `multiple`, so it looks like the parens are meant to be there? I might also change the comma after `mutations` to something else (maybe `;` or `:` or something), so that it doesn't read like an argument list.

Reply:
You can also add `#![allow(unused-parens)]` inside the gen_lock function declaration in the macro to silence the warning

Reply:
> Maybe remove the comma after `multiple`, so it looks like the parens are meant to be there? I might also change the comma after `mutations` to something else (maybe `;` or `:` or something), so that it doesn't read like an argument list.

Remove the comma won't take effect, I have to use `#![allow(unused_parens)]`, but it do makes the syntax look better.